core/src/main/java/com/linecorp/armeria/server/Server.java

```diff
@@ -85,6 +86,9 @@
     @Nullable
     private final DomainNameMapping<SslContext> sslContexts;
 
+    // use for testing purpose only
+    private ServerBootstrap serverBootstrap;
```

Comment:
add that is not pretty, but don't know how to test without exposing bootstrap object itself.